### DISCUSSION TITLE
使用UCSCXenaShiny包的GETX与TCGA对比正常组织和癌症组织的基因表达差异时发现常见的内参基因也有变化

### DISCUSSION THREAD

**Discussion by shaonianabin666** [ShixiangWang/self-study#119] [Q&A] [answered]
使用代码vis_toil_TvsN_cancer()查询精原细胞瘤中基因变化时发现常用的内参基因GAPDH有显著变化，按照常理来讲GAPDH在正常组织和癌症组织之间不是应该变化很小的吗，是不是我哪里搞错了，虚心求教，谢谢大佬们代码如下

vis_toil_TvsN_cancer(
  Gene = "GAPDH",
  Mode = c("Violinplot", "Dotplot"),
  data_type = "mRNA",
  Show.P.value = TRUE,
  Show.P.label = TRUE,
  Method = "wilcox.test",
  values = c("#DF2020", "#DDDF21"),
  TCGA.only = FALSE,
  Cancer = "TGCT",
  opt_pancan = .opt_pancan
)

**Comment by ShixiangWang** [comment, marked as answer]
常用的内参基因的表达本身都会有比较大的变化

https://www.yeasen.com/solutiondetail/64

**Reply by shaonianabin666** [reply]
太感谢老师了！！！！！！！
